css/properties/d.json

```diff
@@ -23,7 +23,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "preview"
+              "version_added": "18"
```

Comment:
This is incorrect. This should be "false". This shows up in the collector because the property parses, but has no functionality.

Reply:
Thanks Jon! Updated to false.